code_of_conduct.md

```diff
@@ -0,0 +1,71 @@
+# LDK Code of Conduct
+
+The LDK community is open to pretty much anyone. To ensure that project forums such as the Discord server and GitHub repository are open and friendly, we count on maintainers and project representatives to behave in a way that is not disruptive to LDK's culture or to any one participant's well-being. 
+
+Therefore, we came up with some easy to follow guidelines.
+ * Be friendly. Interact in a way that fosters openness, inclusivity, and collaboration.   
+ * Be respectful. We may disagree, but disagreement is no excuse for rude behavior or personal attacks.
+ * Be considerate. Provide and accept constructive criticism. 
+ 
+Private or public harassment of any kind will not be tolerated. Since harassment can take many forms, here's a non-exhaustive list of what we consider unacceptable behavior:
+ * Offensive language directed at individuals or groups of people
+ * Bullying (verbal, physical, social, or cyber) 
+ * Interfering with someone's ability to contribute, like with excessive nitpicking 
+ * Continued one-on-one communication after a party has requested it cease
+ * Stalking online or offline
+ * Doxing or unauthorized publication of private information or communication
+ * Unwelcomed sexual attention
+ * Inappropriate visual displays such as sexually-oriented or offensive  photography, cartoons, drawings, or gestures
+ * Retaliation for reporting or threatening to report harassment
```

Comment:
Should we add something -- not necessarily here -- for obvious spam? We had one incident of spam on Discord where deleting posts and banning was the clear action to take. Discord also has a kick and a timeout feature, FWIW.